cfpb/consumerfinance.gov
# Filters: Swap authors filter with a languages filter

Get rid of the authors filter and replace it with a languages filter.

## Removals

- Remove link from author byline on posts.


## Changes

- Get rid of the authors filter and replace it with a languages filter.


## How to test this PR

1. Pull branch and run a local server.
2. Language filter should appear and be functional on any filter with a Topics filter:

http://localhost:8000/activity-log/
http://localhost:8000/about-us/newsroom/
http://localhost:8000/about-us/blog/
http://localhost:8000/administrative-adjudication-proceedings/administrative-adjudication-docket/
http://localhost:8000/compliance/supervisory-guidance/
http://localhost:8000/data-research/research-reports/
http://localhost:8000/rules-policy/final-rules/
http://localhost:8000/consumer-tools/everyone-has-a-story/
http://localhost:8000/rules-policy/rules-under-development/
http://localhost:8000/data-research/cfpb-researchers/
http://localhost:8000/consumer-tools/credit-reports-and-scores/consumer-reporting-companies/companies-list/
http://localhost:8000/about-us/events/
http://localhost:8000/rules-policy/notice-opportunities-comment/open-notices/
http://localhost:8000/rules-policy/notice-opportunities-comment/archive-closed/
http://localhost:8000/compliance/amicus/briefs/
http://localhost:8000/about-us/events/archive-past-events/


## Screenshots

<img width="822" alt="Screen Shot 2021-07-02 at 3 00 45 PM" src="https://user-images.githubusercontent.com/704760/124318240-57132c00-db46-11eb-880a-912645a84caa.png">
